paimon-common/pom.xml

```diff
@@ -198,6 +198,12 @@ under the License.
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-math3</artifactId>
```

Comment:
Please remove this dep

Reply:
I moved some metrics classes from the core to the common project, and the class DescriptiveStatisticsHistogram depends on this.

`package org.apache.paimon.metrics;


import org.apache.commons.math3.stat.descriptive.DescriptiveStatistics;


import java.io.Serializable;


/**
 * The {@link DescriptiveStatisticsHistogram} use a {@link DescriptiveStatistics} as a Paimon {@link
 * Histogram}.
 */
public class DescriptiveStatisticsHistogram implements Histogram, Serializable {`

Reply:
> Please remove this dep

I moved some metrics classes from paimon-core to paimon-common project,  and the class DescriptiveStatisticsHistogram depends on this.